kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -9902,7 +9902,7 @@ messages:
       return;
    }
 
-   BreakTrance(event = 0, what = $)
+   BreakTrance(event = 0, what = $, usewhat = $)
```

Comment:
This is super hard to understand.  What is "what" and "usewhat"?  Add a string comment explaining the arguments.

Reply:
'what' in this context was always associated with EVENT_CAST.  As such I have changed the 'what' argument to 'castwhat' everywhere.  This did necessitate changing some additional files because we check 'what' more than we check the other arguments.  The additional changes are very minor.